src/components/TodoComponent/TodoComponent.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  id: number;
+  title: string;
+  completed: boolean;
+  isSelectedTodo: boolean;
+  onTodoSelected: (todo: Todo) => void;
+};
+
+export const TodoComponent: React.FC<Props> = ({
+  todo,
+  id,
+  title,
+  completed,
+  isSelectedTodo,
+  onTodoSelected,
+}) => {
+  return (
+    <tr
```

Comment:
The 'key' prop should be added when rendering a list of components, not inside the component itself. Remove the 'key' prop from here and add it when rendering a list of TodoComponents. 🗝️

Reply:
So i dont need it when in my situation? Because i dont have any list

Reply:
1) You should add a key while you mapping at TodoList, not inside TodoComponent
2) as a recommendation - change a name from `TodoComponent` to `Todo`. The word `Component` is redundant